certified-connectors/Converter/apiDefinition.swagger.json

```diff
@@ -0,0 +1,204 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Converter",
+    "description": "Enables conversion of different data formats from one to another.",
+    "version": "1.0",
+    "contact": {
+      "name": "Power2Apps Support",
+      "url": "https://www.power2apps.de/kontakt",
+      "email": "kontakt@power2apps.de"
+    }
+  },
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://www.power2apps.de"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://www.power2apps.de/datenschutz"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Content and Files;Data"
+    }
+  ],
+  "host": "webapiconverterservice20211014190508.azurewebsites.net",
+  "basePath": "/Converter",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/ConvertCsvdataToExcel": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {}
+            }
+          }
+        },
+        "operationId": "ConvertCsvdataToExcel",
+        "consumes": [
+          "text/plain"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "summary": "Convert CSV To Excel",
+        "description": "Convert CSV-Data to Excel-File",
+        "x-ms-visibility": "important",
+        "parameters": [
+          {
+            "name": "CSV-Data",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "string",
+              "x-ms-visibility": "important",
+              "title": ""
```

Comment:
@stsiek Empty property values can be removed until unless you have a specific reason to keep them.